tests/client/core/components/TestPaginate.js

```diff
@@ -177,5 +153,47 @@ describe('<Paginate />', () => {
         );
       });
     });
+
+    describe('when the link is to the current page', () => {
```

Comment:
I moved it here because the setup is the same as another section but is quite verbose... but I'll refactor the setup a bit and change it back.